defaults/main.yml

```diff
@@ -11,6 +12,7 @@ django_stack_gcorn_port: 8000
 django_stack_gcorn_log: /var/log/{{django_stack_app_name}}/app
 django_stack_gcorn_app: freedompress.wsgi
 django_stack_gcorn_app_settings: freedompress.settings.production
```

Comment:
Seems like these vars would be better structured as `{{ django_stack_app_name }}.wsgi` and `{{ django_stack_app_name }}.settings.production`. Might still require overrides, but makes more sense to infer namespace.